src/GpuLidarSensor.cc

```diff
@@ -71,6 +71,9 @@ class ignition::sensors::GpuLidarSensorPrivate
 
   /// \brief Publisher for the publish point cloud message.
   public: transport::Node::Publisher pointPub;
+
+  /// \brief Scanning pattern.
+  public: std::string scanningPattern = "avia";
```

Comment:
This should be the enum from gz-rendering/BaseGpuRays.hh.
We should also create a constexpr std::map which maps from std::string -> ScanningPatternEnum.